app/views/quotes/edit.html.erb

```diff
@@ -5,12 +5,18 @@
 <%= render layout: 'layouts/small_width' do %>
   <%= form_for @quote, html: {class: 'space-y-3 space-y'} do |f| %>
     <%= render 'shared/error_messages', object: f.object %>
-    <%= f.label :text %>
-    <%= f.text_area :text, class: input_classes %>
+    <div>
+      <%= f.label 'Quote:' %>
+      <%= f.text_area :text, class: input_classes %>
```

Comment:
De label en de text_area zijn nu niet gelinkt, als iemand ooit blind word is dat heel kut voor zijn screenreader.